datasets/wegenbestand/dataset.json

```
@@ -0,0 +1,55 @@
+{
+  "id": "wegenbestand",
+  "type": "dataset",
+  "authorizationGrantor": "Deze gegevensset wordt onderhouden door bereikbaarheidsthermometer Amsterdam",
```

I think this should be

```suggestion
  "authorizationGrantor": "n.v.t.",
```

because there is no `auth` on this dataset, so there is no need to be granted authorization.